lib/elixir_newbie_web/components/blog_card.ex

```diff
@@ -12,12 +12,13 @@ defmodule ElixirNewbieWeb.Components.BlogCard do
        to={Routes.live_path(ElixirNewbieWeb.Endpoint, BlogShow, @blog.id)}
      >
        <article class="text-white">
-          <figure class="overflow-hidden h-90">
-            <img class="w-auto rounded-lg" src={Routes.static_path(ElixirNewbieWeb.Endpoint, "/images/posts/#{@blog.cover_image}")}/>
-          </figure>
-          <p class="mt-6 text-2xl leading-relaxed capitalize">{@blog.title}</p>
-          <p class="mt-2 text-base leading-relaxed capitalize">{@blog.description}</p>
-          <p class="mt-2 text-gray-300">{Calendar.strftime(NaiveDateTime.new!(@blog.date, Time.utc_now()), "%B %d %Y")}</p>
+        <figure class="overflow-hidden h-90">
+        <img class="w-auto rounded-lg" src={Routes.static_path(ElixirNewbieWeb.Endpoint, "/images/posts/#{@blog.cover_image}")}/>
+        </figure>
+        <p class="mt-6 text-2xl leading-relaxed capitalize">{@blog.title}</p>
+        <p class="mt-2 text-base leading-relaxed capitalize">{@blog.description}</p>
+        <p class="mt-2 text-gray-300">{@blog.author}</p>
```

Comment:
We have a very rough design for the UI https://www.figma.com/file/hcUAM8jBkt45KO8Emx97BL/Elixir-Newbie-Website?node-id=0%3A1

I think yours looks good, though! I'd still accept the PR with this, and I can always update the UI. If you have time, it would be awesome for the name to be on the same line and to the right of the date (see the design)

Reply:
I will change the position